test/cri-containerd/helpers/log.go

```diff
@@ -0,0 +1,61 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"io"
+	"net"
+	"os"
+	"sync"
+
+	"github.com/Microsoft/go-winio"
+	"github.com/pkg/errors"
+)
+
+func main() {
+	var err error
+
+	defer func() {
+		if err != nil {
+			fmt.Fprintln(os.Stderr, err)
+			os.Exit(1)
+		}
+	}()
+
+	ctx, cancel := context.WithCancel(context.Background())
+	defer cancel()
+
+	var sout, wait net.Conn
+
+	soutPipe := os.Getenv("CONTAINER_STDOUT")
+	waitPipe := os.Getenv("CONTAINER_WAIT")
+
+	if sout, err = winio.DialPipeContext(ctx, soutPipe); err != nil {
+		err = errors.Wrap(err, "couldn't open stdout pipe")
+		return
+	}
+	defer sout.Close()
+
+	var dest *os.File
+	destPath := os.Args[1]
+	if dest, err = os.Create(destPath); err != nil {
+		err = errors.Wrap(err, "couldn't open destination file")
+		return
+	}
+	defer dest.Close()
+
+	if wait, err = winio.DialPipeContext(ctx, waitPipe); err != nil {
+		err = errors.Wrap(err, "couldn't open wait pipe")
+		return
+	}
+	wait.Close()
```

Comment:
Could we add a comment here just to say something like "indicate that logging binary is ready to receive output" or something just for future readers?